Traces: Span filtering (#65725)
grafana/grafana: docs/sources/datasources/jaeger/_index.md

}
```

## Span Filters

> **Note:** This feature is behind the `newTraceView` [feature toggle]({{< relref "../../setup-grafana/configure-grafana#feature_toggles" >}}).
> If you use Grafana Cloud, open a [support ticket in the Cloud Portal](/profile/org#support) to access this feature.

![Screenshot of span filtering](/media/docs/tempo/screenshot-grafana-tempo-span-filters.png)

> Allows you to filter your spans in the trace timeline viewer. The more filters you add, the more specific are the filtered spans.

Reviewer: ```suggestion
Using span filters, you can filter your spans in the trace timeline viewer. The more filters you add, the more specific are the filtered spans.
```
